infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -200,7 +200,8 @@ private Container newContainer(
             "--cacert",
             certProvisioner.isConfigured() ? certProvisioner.getCertPath() : "",
             "--registry-address",
-            Strings.nullToEmpty(pluginRegistryUrl)));
+            Strings.nullToEmpty(pluginRegistryUrl),
+            "--merge-plugins"));
```

Comment:
@amisevsk Did not we agree to have some devfile attribute to enable/disable that behavior?
Or it's pretty safe to always enable it since we don't have many such mergeable plugins?

Reply:
It's a good point, ideally we'd have a way to disable the functionality in the devfile, but it would mean adding yet another attribute to the devfile. I can look into adding it.

Reply:
Done.